app/backend/resources/schema.edn

```diff
@@ -1817,7 +1817,88 @@
           {:db/ident :land-owner-opinion/authority-position
            :db/valueType :db.type/string
            :db/cardinality :db.cardinality/one
-           :db/doc "Rich text containing the authoritys position on the opinion"}]]}]
+           :db/doc "Rich text containing the authoritys position on the opinion"}]]}
+
+ {:db/ident :migration/thk-contracts
+  :txes [[{:db/ident :thk.contract/number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to contract file in document management system for procurement part"}
+
+          {:db/ident :thk.contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Procurement no /Hanke nr - (if starts with number then converted into link to State procurement registry as https://riigihanked.riik.ee/rhr-web/#/procurement/<proc_No>), if starts with other symbol then leave it as a number)"}
+
+          {:db/ident :thk.contract/status
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Calculated contract status"}
+
+          ;; THK Regions
+          {:db/ident :thk.region/center
+           :thk/code "Keskus" :enum/attribute :thk.contract/region}
+          {:db/ident :thk.region/north
+           :thk/code "Põhja" :enum/attribute :thk.contract/region}
+          {:db/ident :thk.region/south
+           :thk/code "Lõuna" :enum/attribute :thk.contract/region}
+          {:db/ident :thk.region/west
+           :thk/code "Lääne" :enum/attribute :thk.contract/region}
+          {:db/ident :thk.region/east
+           :thk/code "Ida" :enum/attribute :thk.contract/region}
+
+          {:db/ident :thk.contract/region
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "The region of the Project is displayed"}
+
+          ;; Contract types
+          {:db/ident :thk.contract.type/construction-works
+           :thk/code "3500" :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/things
+           :thk/code "3501" :enum/attribute :thk.contract/type}
+          {:db/ident :thk.contract.type/services
+           :thk/code "3502" :enum/attribute :thk.contract/type}
+
+          {:db/ident :thk.contract/type
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Comes from THK, not editable (activity_procurementType, activity_procurementTypeFK"}
+
+          {:db/ident :thk.contract/signed-at
+           :db/valueType :db.type/instant
+           :db/cardinality :db.cardinality/one
+           :db/doc "Then contract signed, added manually in the detail view of contract"}
+
+          {:db/ident :thk.contract/deadline
```

Comment:
Still needs extended deadline, warranty period and if we want to add the calculated warranty end time as well that could be added.